src/organizers/organizer-manager.service.js

```diff
@@ -32,7 +32,7 @@ function OrganizerManager(udbApi) {
    * @return {Promise.<PagedCollection>}
    */
   service.find = function(query, limit, start) {
-    return udbApi.findOrganisations(start, limit, null, query, true);
+    return udbApi.findOrganisations(start, limit, null, query, false);
```

Comment:
This boolean (useAdvancedQuery) effects the code below:

`if (useAdvancedQuery && query) {
      params.q = query;
  } else if (query) {
      params.name = query;
  }`

found in `udb-api.service.js`

Reply:
Thanks for the explanation 👍 
Do we still use this function with `useAdvancedQuery` set to `true`? If not we would remove that parameter and the if/else and just always use `params.name`